app/android/build.gradle

```diff
@@ -25,7 +19,7 @@ buildscript {
         classpath("com.facebook.react:react-native-gradle-plugin")
         classpath("de.undercouch:gradle-download-task:5.0.1")
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
-        classpath("com.google.gms:google-services:4.3.15")
+        classpath("com.google.gms:google-services:4.3.0")
```

Comment:
We want to downgrade from 4.3.15 to 4.3.0?

Reply:
hmm, let me test.... I believe I did that to limit issues with overlapping dependencies...